### ISSUE TITLE
[Network] - Setting widthConstraint.maximum to node causes error

### ISSUE THREAD

**Issue by Yuvaleros** [almende/vis#2604]
**version: 4.18.0**
When trying to add the widthConstraint maximum attribute I receive an error in the console:
"Uncaught TypeError: Cannot read property 'width' of undefined"

Works fine with widthConstraint.minimum.
 ```
   var nodes = new vis.DataSet([
        {id: 1, label: 'Node 1', shape:'circle', widthConstraint : { maximum : 30}}
    ]);
```

**Comment by mojoaxel** [comment]
@eymiha Can you take a look, please. Thx!

**Comment by eymiha** [comment]
It's an elusive text-formatting edge case!

The error is because 30px is not wide enough to contain the word 'Node' at the default point size. The fix is to always include the first word on each line irrespective of it's width. Otherwise we'd get into an infinite loop.

A fix will be PR'ed momentarily.

**Comment by Yuvaleros** [comment]
Thank you very much!

**Comment by mojoaxel** [comment]
@eymiha provided a fix with ~~~#2606~~~ #2650
